src/components/TextField/TextField.tsx

```diff
@@ -1,27 +1,32 @@
 import styles from './TextField.module.scss';
+import * as React from 'react';
```

Comment:
You should _never_ use a wildcard import for `react` exports, as that will import _literally everything_ from React into this one file. This might not be too big of an issue in modern React, but I would still advise you to narrow your imports down to only what you need (i.e., `useEffect`, `useState`, etc.).